### ISSUE TITLE
Skyrim Sewers 4

### ISSUE THREAD

**Issue by Crowlikedream** [tiltedphoques/Mod-Compatibility#119]
# Mod information

Adds sewer systems to Solitude, Whiterun & Windhelm and a couple of forts.

#### Download link

https://www.nexusmods.com/skyrimspecialedition/mods/9320

#### Dependencies

This mod does not have any known dependencies other than the base game.

# Compatibility

This mod does not work when starting the main game. Game starts, however audio continues as horse, wagon, npcs, etc. are separated and frozen.

**Comment by VENDETTA5** [comment]
My game starts, though I use alternate start and I can even play for an hour + with any non modded player without any problems though I haven't visited sewers yet.

**Comment by Crowlikedream** [comment]
Yeah the problem is starting the main game. The screen freezes at the beginning of the ride down to Helgen. A shame as we wanted to have the sewers in place for our gameplay and role play.

**Comment by Dragonissa** [comment]
Are you sure its actually caused by the mod? Its just a simple creationkit mod, that shouldnt cause such issues.

**Comment by Astral3** [comment]
...To confirm, you were not connected to each other when starting the game, correct?

**Comment by Crowlikedream** [comment]
Each of us was starting separately to create a character and get pass Helgen before connecting. Froze up at the very start.

Troubleshooted it was this mod was causing the issue. Take the mod out can play with no issues. Put the mod in and freezes. Even by itself. We have been testing each mod we have been wanting to use, going through our list one by one so we can get off to a good start and stay that way to when we decide we reach the end.

**Comment by PopulusRegis** [comment]
If you enable the mod after Helgen, it is fine. Haven't tried it with others yet, though.
